### ISSUE TITLE
Random Error, Add Testing for random

### ISSUE THREAD

**Issue by njdaniel** [njdaniel/dnd#8]
error: 
```
panic: invalid argument to Intn

goroutine 1 [running]:
math/rand.(*Rand).Intn(0xc00009c150, 0xffffffffffffffff, 0xb1ba40)
	/usr/local/go/src/math/rand/rand.go:169 +0x9c
math/rand.Intn(...)
	/usr/local/go/src/math/rand/rand.go:337
main.Roll(0xffffffffffffffff, 0xc000187788)
	/home/nick/dev/github.com/njdaniel/dnd/services/character/main.go:700 +0x9c
main.NewCharacter(0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/home/nick/dev/github.com/njdaniel/dnd/services/character/main.go:659 +0x1104
main.main()
	/home/nick/dev/github.com/njdaniel/dnd/services/character/main.go:31 +0x9b

Process finished with exit code 2
```

Version: commitSha bdb97fcaf02dc3db9db7b6f97f16dfaca8082a19

Want: Testing for Testing Random tables. Manually run the multiple times, need to run ~100 to check the edge cases.

**Comment by njdaniel** [comment]
fix error, ed75af523d1183ef6be0b8ba6a6a4c0653921252 catch rolling a negative die

**Comment by njdaniel** [comment]
error fixed, close and will open new issue for tracking testing
